Keas.Mvc/Services/RolesSessionsManager.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Serilog;
+
+namespace Keas.Mvc.Services
+{
+    public interface IRolesSessionsManager

```

Comment:
Something drastic w/ naming here -- how about just `IRolesManager`?  The interface shouldn't care if the implementation is actually done via session or cache or cookies or w/e.  Then we'd want to change the `ClearSessionRoles` method to just `ClearRoles` so it's more generic.

Reply:
Yeah, this is the right direction.